File: save_to_random_file.py
import random
import string
import os
import shutil

# Read the selenium test content
with open('/workspace/selenium_google_test.py', 'r') as f:
    content = f.read()

# Generate a random filename
random_chars = ''.join(random.choices(string.ascii_lowercase + string.digits, k=10))
random_filename = f"selenium_test_{random_chars}.py"

# Write the content to the new file
with open(f'/workspace/{random_filename}', 'w') as f:
    f.write(content)

print(f"Saved Selenium test to {random_filename}")

File: save_to_random_file.js
const { v4: uuidv4 } = require('uuid');
const { filterEvenNumbers } = require('./filter_even.js'); // Import the function

// Example array
const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];

// Filter the even numbers
const evenNumbers = filterEvenNumbers(numbers);

// Generate a random filename using UUID
const randomFileName = `/workspace/output_${uuidv4()}.txt`;

// Convert the result to a string and write to the file
const fs = require('fs');
fs.writeFileSync(randomFileName, JSON.stringify(evenNumbers, null, 2));

console.log(`Filtered even numbers: ${JSON.stringify(evenNumbers)}`);
console.log(`Saved to file: ${randomFileName}`);
